packages/charts/src/charts/bubble/tooltip-content.component.tsx

```diff
@@ -36,18 +41,21 @@ export const TooltipContent: FC<Props> = ({
         title: selectorData[labelSelector],
         color,
         correlations: Object.keys(selectorData).reduce((acc, keyName) => {
+          let value = selectorData[keyName];
+
+          if (formatValue?.xKey && keyName === xDomainKey)
```

Comment:
What do You think about something like below:
```
const formatter = formatValue && formatValue[keyName];
const value = formatter ? formatter(selectorData[keyName]) : selectorData[keyName];
```
It should reduce a multiple conditionals.